package.json

```diff
@@ -17,27 +17,27 @@
   "dependencies": {
     "@react-native-async-storage/async-storage": "~1.15.0",
     "@react-native-community/datetimepicker": "3.5.2",
-    "@react-native-community/slider": "4.1.7",
+    "@react-native-community/slider": "4.2.1",
     "@storybook/addon-actions": "~6.3.0",
     "@storybook/addon-controls": "~6.3.0",
     "@storybook/addon-ondevice-actions": "^6.0.1-alpha.7",
     "@storybook/addon-ondevice-backgrounds": "^6.0.1-alpha.7",
     "@storybook/addon-ondevice-controls": "^6.0.1-alpha.7",
     "@storybook/addon-ondevice-notes": "^6.0.1-alpha.7",
     "@storybook/react-native": "^6.0.1-alpha.7",
-    "expo": "~43.0.2",
+    "expo": "~44.0.6",
     "expo-status-bar": "~1.1.0",
-    "react": "17.0.1",
+    "react": "17.0.2",
     "react-dom": "^17.0.2",
-    "react-native": "0.64.3",
+    "react-native": "^0.68.0",
     "react-native-web": "^0.17.5"
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
     "@storybook/addon-essentials": "~6.3.0",
     "@storybook/addon-links": "~6.3.0",
     "@storybook/addon-react-native-web": "^0.0.14",
-    "@storybook/react": "~6.3.0",
+    "@storybook/react": "~6.4.20",
```

Comment:
React native storybook doesn't support 6.4 yet, It's still being worked on. Could you revert this line?